src/components/List.css

```diff
@@ -13,3 +13,13 @@
 .inactive {
   border-left: 5px solid lightgrey;
 }
+
+/* .flex-container {
```

Comment:
I think you need to un-comment these. : )

Reply:
😆  You are absolutely correct. I think I need to step away from the screen for a bit